.circleci/config.yml

```diff
@@ -97,12 +97,12 @@ jobs:
               # push on master or git tag
               if [ "${CIRCLE_BRANCH}" == "master" ] || [ -n "${CIRCLE_TAG}" ]; then
                 echo "${DOCKER_PASS}" | docker login -u="${DOCKER_USER}" --password-stdin
-                docker tag "mozilla/location" "mozilla/location:${DOCKER_TAG}"
+                docker tag "ichnaea_node:latest" "mozilla/location:${DOCKER_TAG}"
```

Comment:
`ichnaea_node` is a node.js container that builds static assets. I think you want `local/ichnaea_app`:

```
saturn5 ~/mozilla/ichnaea> docker image ls | grep ichnaea
(M=dd51d) saturn5 ~/mozilla/ichnaea> docker image ls | grep ichnaea
ichnaea_mysql                                       latest               79a670413fb0        2 minutes ago       373MB
ichnaea_node                                        latest               bf57fa1aeba7        3 minutes ago       427MB
local/ichnaea_app                                   latest               40b68da9af30        4 minutes ago       959MB
```

Reply:
Nice catch! I was looking at the last line of ``make build``, I should have read higher.